models/spleen_deepedit_annotation/configs/inference.json

```diff
@@ -60,49 +61,79 @@
         "res_block": true
     },
     "network": "$@network_def.to(@device)",
+    "preprocessing_transforms": [
+        {
+            "_target_": "LoadImaged",
+            "keys": "image",
+            "reader": "ITKReader"
```

Comment:
Hi @diazandr3s @tangy5 @yiheng-wang-nv ,

Just curious, may I know why we hard-code to use "ITKReader" here? Default to `None` doesn't work?

Thanks.

Reply:
Hi @Nic-Ma , not sure on my side, I'm also curious about why we need itkreader.

Reply:
This DeepEdit version works specifically on 3D images (CT and MR images). I've found a significant amount of NIfTI images with corrupted headers - they don't work with other readers. 
We can change the reader, this is just a heads-up if we change the reader type.

Reply:
OK, I see, let's keep it for now. thanks.